addusers.py

```diff
@@ -285,7 +256,13 @@ def parse_rows(rows):
     return projects, bad_rows
 
 if __name__ == "__main__":
-    openstack = Openstack(admin_user, admin_pwd, admin_tenant, auth_url, nova_version)
+    auth = v3.Password(auth_url=auth_url,
+                       username=admin_user,
+                       user_domain_id = 'default',
+                       password=admin_pwd)
```

Comment:
I tested this a little bit, it works with expected results both with and without specifying the project_name + project_domain_id.   I thought maybe scoping the token might reduce the number of Keystone API calls but it doesn't seem to have much effect. (19 GET, 7 POST vs. 19 GET, 6 POST).

Maybe @knikolla can shed some light on this?  Would it be more efficient in some other way to get a project-scoped token?